src/Illuminate/Support/Stringable.php

```diff
@@ -67,11 +67,12 @@ public function append(...$values)
     /**
      * Append a new line to the string.
      *
+     * @param  int  $count
      * @return $this
      */
-    public function newLine()
+    public function newLine($count = 1)
```

Comment:
this is a major breaking change to a non-private method in a non-final class.

Reply:
This is a new method that was added a few hours ago. If I am not mistaken there wasn't any release since then so there should be nothing to break here.